gnosis/eth/tests/oracles/test_uniswap_v3.py

```diff
@@ -64,10 +64,19 @@ def test_get_price(self):
         with self.assertRaisesMessage(
             CannotGetPriceFromOracle,
             f"Not enough liquidity on uniswap v3 for pair token_1={s_usd_token_mainnet_address} "
-            f"token_2=0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2",
+            f"token_2=0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2, at least 2 units of each token are required",
         ):
             uniswap_v3_oracle.get_price(s_usd_token_mainnet_address)
 
+        # Test token with no liquidity
+        wrapped_nxm_token_mainnet_address = "0x57Ab1ec28D129707052df4dF418D58a2D46d5f51"
```

Comment:
wNMX address is "0x0d438F3b5175Bebc262bF23753C1E53d03432bDE"

Reply:
Done